tracks/ruby/exercises/leap/readme.md

```diff
@@ -0,0 +1,30 @@
+_Leap_ is one of the first side exercises, unlocked by _TwoFer_. 
+
+## Reasonable Solutions
+
+```ruby
+class Leap
+  year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)
```

Comment:
I think we're missing the method definition here!

Reply:
Thanks for catching that!!